sentinel-dashboard/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM amd64/buildpack-deps:buster-curl as installer
+
+ARG SENTINEL_VERSION
+ARG HOT_FIX_FLAG=""
+
+RUN set -x \
+    && curl -SL --output /var/tmp/sentinel-dashboard.jar https://github.com/alibaba/Sentinel/releases/download/${SENTINEL_VERSION}/sentinel-dashboard-${SENTINEL_VERSION}.jar \
+    && cp /var/tmp/sentinel-dashboard.jar /home \
+    && rm -rf /var/tmp/sentinel-dashboard.jar
```

Comment:
output to /home/sentinel-dashboard.jar maybe more suitable.

Reply:
Done, simplified.